packages/functional-tests/tests/settings/changeEmailBlocked.spec.ts

```diff
@@ -114,14 +115,11 @@ async function changePrimaryEmail(
   );
 }
 
-async function removeAccount(
-  settings: SettingsPage,
-  deleteAccount: DeleteAccountPage,
-  page: Page,
-  password: string
+async function unblockAccount(
+  blockedEmail: string,
+  target: BaseTarget,
+  signinUnblock: SigninUnblockPage
 ) {
-  await settings.deleteAccountButton.click();
-  await deleteAccount.deleteAccount(password);
-
-  await expect(page.getByText('Account deleted successfully')).toBeVisible();
+  const code = await target.emailClient.getUnblockCode(blockedEmail);
+  await signinUnblock.fillOutCodeForm(code);
```

Comment:
Feels odd to not have an assertion after we do this action. Maybe add something like the following to make sure the action completed properly before moving to teardown.

```
      await expect(settings.settingsHeading).toBeVisible();
```

Reply:
This is intentional, because the result is not always the same depending on preceding steps (e.g., if the password was incorrect). The assertions after unblock are included in test steps.